src/gui/CMakeLists.txt

```diff
@@ -130,6 +123,9 @@ ecm_add_qml_module(owncloudGui
             qml/AccountButton.qml
             qml/FolderDelegate.qml
             qml/FolderError.qml
+
+            spaces/qml/SpaceDelegate.qml
```

Comment:
Adding the qml file in the subdirectory failed during the cache generation so I had to move it here.